src/arrayMethodJoin.js

```diff
@@ -5,7 +5,26 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
-    // write code here
+    let result = '';
+    let char = String(separator);
+
+    if (separator === undefined) {
+      char = ',';
```

Comment:
you can use it in default

Reply:
agree, set coma as default value for separator in function params